test/sqllogictest/unsigned_int.slt

```diff
@@ -763,3 +764,252 @@ query T
 SELECT 15445::uint8::text
 ----
 15445
+
```

Comment:
I'll remove the numeric casts once [#14401](https://github.com/MaterializeInc/materialize/pull/14401) is merged.